### PR TITLE
simpler plugin, hooking straight into the run task.

### PR DESCRIPTION
**Pull request by ivantopo**
I stripped down the plugin to the bare minimum that provides the functionality we want: be able to run applications with the AspectJ Weaver by default. 

The RunWithAspectJ is mostly taken from SBT and tries to keep all functionality that was already there (although, I don't understand much of the reasoning behind those parts of SBT).

### CODE REVIEW

File: examples/ping-pong/src/main/scala/kamon/aspectj/sbt/runner/PingPong.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * =========================================================================================
  */
 
-package kamon.aspectj.sbt.runner
+package kamon.aspectj.sbt.runner2
```
Comment (dpsoft) [inline examples/ping-pong/src/main/scala/kamon/aspectj/sbt/runner/PingPong.scala]: runner2?

**Comment by dpsoft** [comment]
@ivantopo looks good to me if this change supports the reload feature of play. Also i think that we need reflect this in the Readme.

**Review by dpsoft — COMMENTED** [review]
@ivantopo looks good to me if this change supports the reload feature of play. Also i think that we need reflect all changes in the Readme.

File: examples/ping-pong/src/main/scala/kamon/aspectj/sbt/runner/PingPong.scala (hunk 1 above)
Comment (ivantopo) [reply examples/ping-pong/src/main/scala/kamon/aspectj/sbt/runner/PingPong.scala]: Oh, that's a leftover from the refactoring.

**Pull request merged**